screens/Home/MyVcsTab.tsx

```diff
@@ -80,6 +80,13 @@ export const MyVcsTab: React.FC<HomeScreenTabProps> = props => {
   controller.downloadFailedVcs.forEach(vc => {
     failedVCsList.push(`\n${vc.idType}:${vc.id}`);
   });
+
+  const isVerificationError = controller.verificationErrorMessage !== '';
```

Comment:
can we rename it as "isVerificationFailed", since we are checking for presence of error, not getting the error.